config/system.yaml

```
@@ -95,7 +74,11 @@
       type: "turn"
       description: "Correctness vs expected answer using custom LLM evaluation"
       framework: "custom"
-      
+
+    "custom:tool_eval":
+      type: "turn"
+      description: "Tool call evaluation comparing expected vs actual tool calls"
+      framework: "custom"
   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)